MAUI/DigitalGauge/Character-Types.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Character Types in Maui Digital Gauge control | Syncfusion
```

Comment:
Character Types in .NET MAUI Digital Gauge control

**Use** .NET MAUI  in all places